docker-compose.yml

```diff
@@ -60,6 +60,12 @@ services:
     env_file:
       .env
 
+  data-integration:
+    build:
+      context: ./di
```

Comment:
I fthink you could rename the directory data-integration to match the name of the docker image.